### PR TITLE
Bugfixing

### PR DESCRIPTION
**Pull request by wintuxx**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the [**CONTRIBUTING**](Contributing.md) document.

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes to Core Features:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you successfully ran tests with your changes locally?

added all changes from @jakermx and @CastagnaIT

Trying to keep this project alive

### CODE REVIEW

**Comment by wintuxx** [comment]
okay there are many issues to fix. i revolk the pull

**Comment by CastagnaIT** [comment]
My changes do not break functionality it can be added those commits.
The customizations at the views were not working even before.
To fix view settings definitively, i'm working to centralize the configuration of the main menu management to facilitate future changes (views, context, names, etc.) at least in part.
Not knowing the project well, it's not easy to understand everything...

Besides that, are there other serious problems?

**Comment by wintuxx** [comment]
yes i can confirm your changes are working. 
I'm not that familiar with python yet. I let myself be confused by the test.
I open the pull again and then we'll see

**Comment by CastagnaIT** [comment]
Please wait for the last commit is not definitive

File: resources/lib/services/msl/converter.py [hunk 1]
```diff
@@ -20,11 +20,16 @@ def convert_to_dash(manifest):
     root = _mpd_manifest_root(duration)
     period = ET.SubElement(root, 'Period', start='PT0S', duration=duration)
     protection = _protection_info(manifest)
+<<<<<<< HEAD
```
Comment (ccope) [inline resources/lib/services/msl/converter.py]: This file has lots of unresolved merge conflicts

File: resources/lib/services/msl/msl_handler.py [hunk 2]
```diff
@@ -26,9 +26,14 @@
 
 CHROME_BASE_URL = 'https://www.netflix.com/nq/msl_v1/cadmium/'
 ENDPOINTS = {
+<<<<<<< HEAD
```
Comment (ccope) [inline resources/lib/services/msl/msl_handler.py]: This file also has unresolved merge conflicts

File: resources/lib/services/msl/request_builder.py [hunk 3]
```diff
@@ -72,16 +73,21 @@ def _headerdata(self, esn, is_handshake=False, is_key_request=False,
             'handshake': is_handshake,
             'nonreplayable': False,
             'capabilities': {
+<<<<<<< HEAD
```
Comment (ccope) [inline resources/lib/services/msl/request_builder.py]: This file as well.

Comment (wintuxx) [reply resources/lib/services/msl/request_builder.py]: thx for your sugesstions.
I have fixed them all allready

File: resources/lib/services/msl/msl_handler.py (hunk 2 above)
Comment (wintuxx) [reply resources/lib/services/msl/msl_handler.py]: fixed

File: resources/lib/services/msl/converter.py (hunk 1 above)
Comment (wintuxx) [reply resources/lib/services/msl/converter.py]: fixed

**Comment by CastagnaIT** [comment]
I understand the intention to keep caphm up to date, but this is counterproductive, if caphm decides to continue the development he will take the commits directly from my repository not from this mess...

**Pull request closed without merging**